core/Integer.Format.savi

```diff
@@ -8,6 +8,8 @@
   :fun hexadecimal: Integer.Format.Hexadecimal(T)._new(@_value)
   :fun binary: Integer.Format.Binary(T)._new(@_value)
 
```

Comment:
I'm just suggesting here to remove this empty line above, so that it gets grouped with the similar methods above.
```suggestion
```